test/test-data.js

```diff
@@ -186,3 +186,33 @@ it('POST Request /data/acceptedcategories/organization/:id : add new AcceptedCat
     }
   );
 });
+
+it('GET Request /data/organizations/:id : Get an Organization', function (done) {
+  request(`http://localhost:3000/data/organizations/${ORG_FURNITURE_BANK_ID}`, function (
+    error,
+    response,
+    body
+  ) {
+    expect(response.statusCode).to.equal(200);
+    expect(JSON.parse(response.body)).to.deep.equal(ORG_FURNITURE_BANK);
+    done();
+  });
+});
+
+it('POST Request /data/organizations/:id : Update an Organization', function (done) {
+  request.post(
+    {
+      headers: {'content-type': 'application/json'},
+      url: `http://localhost:3000/data/organizations/${ORG_FURNITURE_BANK_ID}`,
```

Comment:
let's use the `BASE_URL` flag in the env file [1] instead of hardcoding localhost:3000.

[1] https://github.com/googleinterns/donatia/blob/master/.env.sample#L10

Same comment goes to the other `localhost:3000` in this file.